usaon_benefit_tool/templates/users.html

```diff
@@ -9,4 +9,15 @@ <h2>{% block title %}Users{% endblock %}</h1>
     show_actions=True,
     edit_url=('user.get', [('user_id',':id')]),
   ) }}
+
+{% block scripts %}
+  {{ super() }}
+
+  <script>
+    jQuery(document).ready( function () {
+        jQuery('table').DataTable();
+    } );
+  </script>
+{% endblock %}
+
```

Comment:
Since this is repeated a few times, what do you think about moving it to a macro?

Reply:
Yes I also thought about this but eventually decided to not put it in macro. I am thinking of adding an additional file in macro for this would that be okay ?

Reply:
That sounds great! Thanks, @Sherwin-14 :)